bluemira/codes/wrapper.py

```diff
@@ -36,6 +38,12 @@ def systems_code_solver(
         build configuration dictionary
     module:
         Module to use
+    params_cls:
+        Optional custom ParameterFrame class
+        that contains a set of parameter mapping to be used
+        in :class:`bluemira.codes.process._solver.Solver`.
+        If not provided, the default one will be used:
+        `bluemira.codes.process.params.ProcessSolverParams
```

Comment:
This is technically not exclusive to PROCESS as the interface is meant to be generic for any systems code.
The ~ makes sure a link to the class is created in the API docs
```suggestion
        A custom :class:`~bluemira.codes.params.MappedParameterFrame`
        to be used by the systems code solver. If not provided, the default 
        one will be used.
```